docs/hooks/template.md

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
Won't let me comment here directly on the line, but change the hook maturity link above to 'current' instead of '1.0'

Reply:
Dennis, this is potentially messy because the HL7 publish will presumably be to 1.1, not current; however, it also looks like this template page isn't curently published at hl7.org, soo .... made this change.